index.js

```diff
@@ -310,6 +327,10 @@ const ContainerService = {
   },
 
   sortTabs() {
+    sendEvent({
```

Comment:
I don't like this :)
I prefer this:

this._sendTelemetryData('container-sort-tabs');

where _sendTelemetryData is:

function _sendTelemetryData(event) {
  let totalContainerTabsCount = 0;
  for ... identitiesState...

  this._sendEvent({
    event,
    "total-container-tabs-count": totalContainerTabsCount,
  });
}